data/modules/PolicePatrol/PolicePatrol.lua

```diff
@@ -169,13 +163,37 @@ local onEnterSystem = function (player)
 			end)
 		end
 	end
+
+	local police = ShipDef[system.faction.policeShip]
+	Timer:CallEvery(15, function ()
+		if not Game.system or #patrol == 0 then return true end
+		if target then return false end
+
+		local ships = Space.GetBodiesNear(patrol[1], lawEnforcedRange, "Ship")
+		for _, enemy in ipairs(ships) do
+			if enemy.shipId ~= police.id and enemy:GetCurrentAICommand() == "CMD_KILL" then
+				if not piracy then
+					Comms.ImportantMessage(string.interp(l_ui_core.X_CANNOT_BE_TOLERATED_HERE, { crime = l_ui_core.PIRACY }, patrol[1].label))
+					Comms.ImportantMessage(string.interp(l["RESTRICTIONS_WITHDRAWN_" .. Engine.rand:Integer(1, getNumberOfFlavours("RESTRICTIONS_WITHDRAWN"))], { ship_label = player.label }, patrol[1].label))
```

Comment:
This code is passing the police ship's label as a third parameter to `string.interp` rather than the `from` parameter to `Comms.ImportantMessage`.